definitions/region/africa-interconnections.yaml

```
@@ -0,0 +1,227 @@
+common_regions:
```

The label common_regions is usually reserved for comparison regions between models. 
I would suggest something more obvious like:

```suggestion
Two country regions:
```